activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -510,16 +510,18 @@ def drop_table(table_name, options = {})
       # agnostic and should usually be avoided.
       #
       # Available options are (none of these exists by default):
-      # * <tt>:limit</tt> -
-      #   Requests a maximum column length. This is the number of characters for a <tt>:string</tt> column
-      #   and number of bytes for <tt>:text</tt>, <tt>:binary</tt> and <tt>:integer</tt> columns.
-      #   This option is ignored by some backends.
+      # * <tt>:limit</tt> - Maximum column length. Ignored by some backends.
+      #   * <tt>:string</tt> column - Number of characters
+      #   * <tt>:text</tt>, <tt>:binary</tt>, <tt>:integer</tt> - Number of bytes
+      #   * <tt>:datetime</tt> - Number of digits of fractional seconds
```

Comment:
Since it's deprecated, I think it's okay to not talking about setting limit on`:datetime` column